xds/internal/balancer/edsbalancer/balancergroup_test.go

```diff
@@ -32,9 +34,21 @@ import (
 var (
 	rrBuilder        = balancer.Get(roundrobin.Name)
 	testBalancerIDs  = []internal.Locality{{Region: "b1"}, {Region: "b2"}, {Region: "b3"}}
-	testBackendAddrs = []resolver.Address{{Addr: "1.1.1.1:1"}, {Addr: "2.2.2.2:2"}, {Addr: "3.3.3.3:3"}, {Addr: "4.4.4.4:4"}}
+	testBackendAddrs []resolver.Address
 )
 
+const testBackendAddrsCount = 8
+
+func init() {
+	for i := 0; i < testBackendAddrsCount; i++ {
+		testBackendAddrs = append(testBackendAddrs, resolver.Address{Addr: fmt.Sprintf("%d.%d.%d.%d:%d", i, i, i, i, i)})
+	}
+
+	// Disable caching for all tests. It will be re-enabled in caching specific
+	// tests.
+	defaultSubBalancerCloseTimeout = time.Millisecond
```

Comment:
I guess this should be fine here because we are overriding it only once. But on the other PR that I was working on, I found a data race when some of the tests were overriding a package level var but some werent. Do you think it would make sense to inject this value into the `balancerGroup` instead of defining it as a package level var. Something to think about, and maybe add a TODO. No need to fix it in this PR.

Reply:
Sure. Even a TODO here and doing it later should be fine. Thanks.